workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs

```diff
@@ -31,10 +33,15 @@ public EntityGameObjectCreator(World world, Dictionary<string, GameObject> initi
                 cachedPrefabs[prefabPath] = prefab;
             }
 
-            var gameObject = GameObject.Instantiate(prefab, position, rotation);
+            var gameObject = Object.Instantiate(prefab, position, rotation);
             gameObject.name = $"{prefab.name}(SpatialOS: {spatialEntityId}, Unity: {entity.Index}/{world.Name})";
             return gameObject;
         }
+
+        public void DeleteGameObject(Entity entity, GameObject gameObject)
```

Comment:
What is the `Entity` pass in for?

Reply:
look at https://github.com/spatialos/UnityGDK/pull/272 where it gets changed to an interface